### PR TITLE
Add option to select type of appointment given

### PR DESCRIPTION
**Pull request by benbarnett**
Update documents templates to version 4.0, including content updates and ability to create 50-54 and 55+ appointment types.

### CODE REVIEW

**Comment by benbarnett** [comment]
@andrewgarner when you have a minute could you give this some :eyes:?

File: spec/models/appointment_summary_spec.rb [hunk 1]
```diff
@@ -58,6 +58,15 @@
     expect(subject.format_preference).to eq('standard')
   end
 
+  it do
+    is_expected
+      .to validate_inclusion_of(:appointment_type).in_array(%w(standard 50_54))
+  end
+
+  it 'has a defaults to a standard appointment type' do
```
Comment (andrewgarner) [inline spec/models/appointment_summary_spec.rb]: `it 'defaults to a standard appointment type' do`

Comment (benbarnett) [inline spec/models/appointment_summary_spec.rb]: Oops just spotted that too: https://github.com/guidance-guarantee-programme/summary_generator/blob/master/spec/models/appointment_summary_spec.rb#L57

Comment (benbarnett) [inline spec/models/appointment_summary_spec.rb]: Happy for me to fix both in this PR?

**Comment by andrewgarner** [comment]
You might need to tweak the style guide too. I don't think it's got test coverage though :(

File: spec/models/appointment_summary_spec.rb (hunk 1 above)
Comment (benbarnett) [inline spec/models/appointment_summary_spec.rb]: In fact I'll just fix the other one separately

Comment (andrewgarner) [inline spec/models/appointment_summary_spec.rb]: :+1:

**Comment by benbarnett** [comment]
Yep, updated styleguide

**Comment by andrewgarner** [comment]
:+1:

**Comment by andrewgarner** [comment]
Ruby zen

**Pull request merged**